app/build.gradle

```diff
@@ -90,6 +90,8 @@ dependencies {
   kaptAndroidTest(
       'com.google.dagger:dagger-compiler:2.24'
   )
+  //TODO (#59): Remove this once Bazel is introduced
```

Comment:
Nit: space before TODO.

Ditto elsewhere in this PR.

Reply:
Done.